packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -1013,110 +1013,25 @@ describe('Table Interactive Column Sizing', () => {
-            },
-            {
-                name: 'click on third column right divider',
-                dividerClickIndex: 4,
-                expectedColumnActiveDividerIndexes: [3, 4]
-            },
-            {
-                name: 'click on last column left divider',
-                dividerClickIndex: 5,
-                expectedColumnActiveDividerIndexes: [5]
-            }
-        ] as const;
-        parameterizeSpec(dividerActiveTests, (spec, name, value) => {
-            spec(
-                `${name} updates expected dividers as "divider-active" and "column-active"`,
-                async () => {
-                    const dividers = Array.from(
-                        element.shadowRoot!.querySelectorAll('.column-divider')
-                    );
-                    const divider = dividers[value.dividerClickIndex]!;
-                    const dividerRect = divider.getBoundingClientRect();
-                    const mouseDownEvent = new MouseEvent('mousedown', {
-                        clientX: (dividerRect.x + dividerRect.width) / 2,
-                        clientY: (dividerRect.y + dividerRect.height) / 2
-                    });
-                    const mouseUpEvent = new MouseEvent('mouseup');
-                    divider.dispatchEvent(mouseDownEvent);
-                    await waitForUpdatesAsync();
-                    const dividerActiveDividers = [];
-                    const columnActiveDividers = [];
-                    for (let i = 0; i < dividers.length; i++) {
-                        if (dividers[i]!.classList.contains('divider-active')) {
-                            dividerActiveDividers.push(i);
-                        }
-                        if (dividers[i]!.classList.contains('column-active')) {
-                            columnActiveDividers.push(i);
-                        }
-                    }
-                    document.dispatchEvent(mouseUpEvent); // clean up registered event handlers
-
-                    expect(dividerActiveDividers.length).toEqual(1);
-                    expect(dividerActiveDividers[0]).toEqual(
-                        value.dividerClickIndex
-                    );
-                    expect(columnActiveDividers).toEqual(
-                        value.expectedColumnActiveDividerIndexes
-                    );
-                }
-            );
-        });
-
-        it('first column only has right divider', () => {
-            const rightDivider = pageObject.getColumnRightDivider(0);
-            const leftDivider = pageObject.getColumnLeftDivider(0);
-
-            expect(rightDivider).not.toBeNull();
-            expect(leftDivider).toBeNull();
-        });
-
-        it('last column only has left divider', () => {
-            const rightDivider = pageObject.getColumnRightDivider(3);
-            const leftDivider = pageObject.getColumnLeftDivider(3);
+    describe('active divider styling', () => {
```

Comment:
I don't think it's necessary to test every divider in a multi-column table. Also, I think it's fair enough to just test that the styling for a given divider changes when pressed, then again when released.